src/css/main.pcss

```diff
@@ -2,4 +2,6 @@
 
 @import 'tailwindcss/components';
 
-@import 'tailwindcss/utilities';
\ No newline at end of file
+@import 'tailwindcss/utilities';
+
+@import './typography.pcss';
```

Comment:
This should be moved to line 4 - our custom typography classes are component classes, not utility classes.